docker/staging-scheduler/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sqlalchemy
```

Comment:
The setup.py file doesn't specify a version, but it's a bit of a smell IMO that sqlalchemy isn't pinned.